recipes/qgis/recipe.sh

```diff
@@ -8,10 +8,10 @@ DEPS_qgis=(zlib gdal qca libspatialite libspatialindex expat gsl postgresql libz
 # DEPS_qgis=()
 
 # url of the package
-URL_qgis=https://github.com/qgis/QGIS/archive/df28e637cdddc0c6e93355a9f98ad2d12d9e6acc.tar.gz
+URL_qgis=https://github.com/qgis/QGIS/archive/94ac9f21b8b7e36d3aa9715922ad15a7bf62fdae.tar.gz
```

Comment:
Which commit is that?
I assumed we either add a patch or bump the version.

Reply:
I did an update to QGIS 3.16.3 to grap latest fixes and optimizations, on top of which we apply the patch.

Reply:
Ah cool. Can we use the branch name instrad of the sha?

Reply:
Sounds like a good idea. TIL we can do that :)